spec/models/auth_token_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe AuthToken do
+  it 'encodes and decodes tokens' do
```

Comment:
These should be two separate tests, as they are testing two cases. If your encode fails, you won't know if the decode fails.